packages/cli-repl/src/arg-mapper.ts

```diff
@@ -93,4 +91,27 @@ export function applyTlsCertificateSelector(
   }
 }
 
+function getCertificateExporter(): TlsCertificateExporter | undefined {
+  if (process.env.TEST_OS_EXPORT_CERTIFICATE_AND_KEY_PATH) {
+    return require(process.env.TEST_OS_EXPORT_CERTIFICATE_AND_KEY_PATH);
+  }
+
+  let packageName: string;
+  switch (process.platform) {
+    case 'win32':
+      packageName = 'win-export-certificate-and-key';
+      break;
+    case 'darwin':
+      packageName = 'macos-export-certificate-and-key';
+      break;
+    default:
+      return undefined;
+  }
+
+  try {
+    return require(packageName);
```

Comment:
Heads up: This is going to break bundling :) The `require()` call needs literals, unfortunately – I had run into this as well :)

Reply:
NOOOOO :D